docs/How-to/Run-a-full-Terra-node/Build-Terra-core.md

```diff
@@ -9,10 +9,21 @@
 
 Use `git` to retrieve [Terra core](https://github.com/terra-money/core/), and checkout the `master` branch, which contains the latest stable release.
 
+For Validators install `v0.x.x-oracle` this will enable mempool prioritization
```

Comment:
If you are a validator, install `v0.x.x-oracle` to enable mempool prioritization. Otherwise, install `v0.x.x`.